python/ray/experimental/serve/tests/test_api.py

```diff
@@ -1,33 +1,25 @@
 import time
 
 import requests
-from flaky import flaky
 
 import ray
 from ray.experimental import serve
 
 
-def delay_rerun(*_):
-    time.sleep(1)
-    return True
-
-
-# flaky test because the routing table might not be populated
-@flaky(rerun_filter=delay_rerun)
 def test_e2e(serve_instance):
     serve.create_endpoint("endpoint", "/api")
     result = ray.get(
         serve.global_state.kv_store_actor_handle.list_service.remote())
     assert result == {"/api": "endpoint"}
 
+    time.sleep(1)
```

Comment:
There should probably be a better way to do this :)

Reply:
Fixed.